.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,3 @@
-**Issue:** https://issues.redhat.com/browse/PROJQUAY-???
-Pull-request title must start with "PROJQUAY-??? - "
+Please follow commit message requirements:
+<scope>: <subject> (PROJQUAY-####)
```

Comment:
We don't really need this template if we're going to enforce a commit message format.
This should be in a README or HOWTO for contributing or something.

Reply:
removed